HW3.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import tempfile
+
+from pyspark.sql import SparkSession
+
+import requests
+from pyspark import StorageLevel
+from pyspark.ml.classification import RandomForestClassifier
+from pyspark.ml.feature import CountVectorizer
+
+
+
+def main():
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    # Load Data directly from MariaDB
+
+    t_rolling_lookup_df = spark.read \
+        .format("jdbc") \
+        .option("url", "jdbc:mysql://localhost:3306/baseball") \
+        .option("driver", "org.mariadb.jdbc.Driver") \
+        .option("query",
+                '''
+                SELECT 
```

Comment:
Nice. I don't know that we can do a SQL query when reading the database like this.